src/renderers/shared/ReactPerf.js

```diff
@@ -13,17 +13,31 @@
 
 var ReactDebugTool = require('ReactDebugTool');
 var warning = require('warning');
+var alreadyWarned = false;
 
 function roundFloat(val, base = 2) {
   var n = Math.pow(10, base);
   return Math.floor(val * n) / n;
 }
 
+function returnWarnIfDevFalse(returningValue = 0) {
+  if (!alreadyWarned) {
+    alreadyWarned = true;
+    warning(__DEV__,
```

Comment:
So, it turns out we can’t use `warning` here because it gets stripped in production 😂 .
Let’s do this instead:

```js
if (typeof console !== 'undefined') {
  console.error(...);
}
```